client/gutenberg/editor/main.jsx

```diff
@@ -36,7 +36,7 @@ class GutenbergEditor extends Component {
 			return null;
 		}
 
-		overrideAPIPaths( this.props.siteSlug );
+		applyMiddlewares( this.props.siteSlug );
```

Comment:
:+1: for the rename, maybe `applyAPIMiddlewares` would be more appropriate?

Reply:
I was thinking on dropping the `utils.js` in favor of `middlewares.js` or something like that, but since naming things is hard, I'll leave it for another PR. Updated on fca7b6135a91b47113af39793ebf0812a14bb32b.